flit/__init__.py

```diff
@@ -58,11 +62,14 @@ def main(argv=None):
     elif args.subcmd == 'install':
         from .install import Installer
         Installer(args.ini_file, user=args.user, symlink=args.symlink).install()
+    elif args.subcmd == 'register':
+        from .wheel import WheelBuilder
+        WheelBuilder(args.ini_file).register()
```

Comment:
This shouldn't be a WheelBuilder method, I don't think. It shouldn't be necessary to do any of the wheel stuff in order to register the package. I'd make a new method in upload that loads the ini and module info and then calls register(). I'm not quite sure what to call the new function, though. Maybe the new one should be 'register' and the existing one `_register_metadata()` or something.